### ISSUE TITLE
Homey or the app crashes

### ISSUE THREAD

**Issue by greetoz** [WeeJeWel/com.mi.flora#2]
This app makes my Homey unstable or it crashes itself. It finds my 3 plant sensors without problem but no data is received. After reboot of homey i get data on one sensor but then homey crashes. When i delete the app homey is stable again.

Sent a log via Homey Settings.

**Comment by WeeJeWel** [comment]
What do you mean with 'Homey crashes'?

**Comment by greetoz** [comment]
I get the notification Homey is Offline and i need to do a ptp.

**Comment by WeeJeWel** [comment]
Can you send me a crashlog after the ptp?

**Comment by greetoz** [comment]
I sent you a new log. ptp was about 30 min ago and did some testing (disabling other apps etc). Seemed more stable when i disabled all other apps. 

 8d8018e9-048d-4be1-aa12-1ac0f235144c log ID

**Comment by WeeJeWel** [comment]
I mean from Homey itself: Settings > System > Send Crash log

**Comment by greetoz** [comment]
ID F2B7CE217C

**Comment by WeeJeWel** [comment]
Thank you, we will investigate

**Comment by jwikman** [comment]
It seems as I have the same issue. I can read the sensor once. On the next try Homey reboots.
I also got a message that the app itself crashed earlier today.

**Comment by jwikman** [comment]
I just disabled the app, which also made Homey crash. Just let me know if you want any more info.

**Comment by r00p** [comment]
Yep app crash everyday, dont know why. Also feels that homey is slow when this app is running. But i am not sure..yet
